upstream-community-operators/hazelcast-enterprise/hazelcast-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    alm-examples: |-
+      [{
+       "apiVersion": "hazelcast.com/v1alpha1",
+       "kind": "Hazelcast",
+       "metadata": {
+          "name": "hz"
+       },
+       "spec": {
+          "image": {
+            "tag": "3.11.2"
+          },
+          "hazelcast": {
+            "licenseKeySecretName": "hz-license-key-secret"
+          },
+          "securityContext": {
+            "runAsUser": 65534,
+            "fsGroup": 65534
+          }
+       }
+      }]
+    categories: "In-Memory Computing Platform"
```

Comment:
Please use categories as listed in this doc: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories